src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonManager.cs

```diff
@@ -103,8 +104,6 @@ internal TimeSpan MinimumLeaseRenewalInterval
 
         public async virtual Task<object> TryLockAsync(string lockId, string functionInstanceId, SingletonAttribute attribute, CancellationToken cancellationToken, bool retry = true)
         {
-            _trace.Verbose(string.Format(CultureInfo.InvariantCulture, "Waiting for Singleton lock ({0})", lockId), source: TraceSource.Execution);
```

Comment:
@mathewc -- I know there's an outstanding question above -- should we add that as an issue to address in the future? Cleaning up these logs now would still be valuable, I think.